src/main/java/thePackmaster/relics/BagOfHolding.java

```diff
@@ -81,5 +90,49 @@ public String getUpdatedDescription() {
         return DESCRIPTIONS[0] + sb + DESCRIPTIONS[1];
     }
 
+    private static final Map<String, Integer> stats = new HashMap<>();
+    public static String ENERGY_STAT = "energy";
+
+    public String getStatsDescription() {
+        if (stats.get(ENERGY_STAT) == null) {
+            return "";
+        }
+        return DESCRIPTIONS[2].replace("{0}", stats.get(ENERGY_STAT) + "");
+    }
+
+    public String getExtendedStatsDescription(int totalCombats, int totalTurns) {
+        if (stats.get(ENERGY_STAT) == null) {
+            return "";
+        }
+        DecimalFormat format = new DecimalFormat("#.###");
+        float block = stats.get(ENERGY_STAT);
+        String energyPerTurn = format.format(block / Math.max(totalTurns, 1));
+        String energyPerCombat = format.format(block / Math.max(totalCombats, 1));
+        return getStatsDescription() + DESCRIPTIONS[3].replace("{0}", energyPerTurn).replace("{1}", energyPerCombat);
+    }
+
+    public void resetStats() {
+        stats.put(ENERGY_STAT, 0);
+    }
+
+    public JsonElement onSaveStats() {
+        Gson gson = new Gson();
+        List<Integer> statsToSave = new ArrayList<>();
+        statsToSave.add(stats.get(ENERGY_STAT));
+        return gson.toJsonTree(statsToSave);
+    }
+
+    public void onLoadStats(JsonElement jsonElement) {
+        if (jsonElement != null) {
+            JsonArray jsonArray = jsonElement.getAsJsonArray();
+            stats.put(ENERGY_STAT, jsonArray.get(0).getAsInt());
+        } else {
+            resetStats();
+        }
+    }
+
+    public static void incrementEnergyStat() {
+        stats.put(ENERGY_STAT, stats.getOrDefault(ENERGY_STAT, 0) + 1);
```

Comment:
This is all written so that even if relic stats isn't loading, everything will be fine -- thus the use of getOrDefault here and elsewhere.